Introduce a header that will contain all backport-specific code and definitions.
Index: src/drivers/net/ethernet/intel/i40evf/i40e_backport_compat.h
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ src/drivers/net/ethernet/intel/i40evf/i40e_backport_compat.h	2018-02-27 18:27:39.893459214 +0100
@@ -0,0 +1,3 @@
+#ifndef I40EVF_BACKPORT_COMPAT_H
+#define I40EVF_BACKPORT_COMPAT_H
+#endif /* I40EVF_BACKPORT_COMPAT_H */
